static/src/components/setting/index.vue

```diff
@@ -220,7 +220,7 @@ export default {
         if (JSON.parse(dataProjConfig.data_access).length > 0) {
             this.dataAccessConfigured = true;
         }
-        if (this.dataAccessConfigured) {
+        if (this.dataAccessConfigured && this.validAccessLevels.length > 0) {
             this.accessLevels = this.getAccessLevels(dataProjConfig.data_access);
```

Comment:
maybe also remove 
`this.accessLevels = this.getAccessLevels(dataProjConfig.data_access);` 
since it is not used anymore. Same as line 132.

Reply:
updated.